utilities/bash/run_six.sh

```diff
@@ -1077,26 +1111,45 @@ function condor_sub(){
 	    done < ${sixdeskjobs}/${LHCDesName}.list
 	else
 	    sixdeskmess -1 "Submission was successful"
-	    # parse terse output (example: "23548.0 - 23548.4")
-	    clusterID=`echo "${terseString}" | head -1 | cut -d\- -f2 | cut -d\. -f1`
-	    clusterID=${clusterID//\ /}
-	    jobIDmax=`echo "${terseString}" | head -1 | cut -d\- -f2 | cut -d\. -f2`
-	    let jobIDmax+=1
-	    nCases=`wc -l ${sixdeskjobs}/${LHCDesName}.list | awk '{print ($1)}'`
-	    if [ ${jobIDmax} -ne ${nCases} ] ; then
-		sixdeskmess -1 "Something wrong with htcondor submission: I requested ${nCases} to be submitted, and only ${jobIDmax} actually made it!"
+	    if [ "$sixdeskplatform" == "htcondor" ]; then
+		# parse terse output (example: "23548.0 - 23548.4")
+		clusterID=`echo "${terseString}" | head -1 | cut -d\- -f2 | cut -d\. -f1`
+		clusterID=${clusterID//\ /}
+		jobIDmax=`echo "${terseString}" | head -1 | cut -d\- -f2 | cut -d\. -f2`
+		let jobIDmax+=1
+		nCases=`wc -l ${sixdeskjobs}/${LHCDesName}.list | awk '{print ($1)}'`
+		if [ ${jobIDmax} -ne ${nCases} ] ; then
+		    sixdeskmess -1 "Something wrong with htcondor submission: I requested ${nCases} to be submitted, and only ${jobIDmax} actually made it!"
+		fi
+		# save taskIDs
+		sixdeskmess -1 "Updating DB..."
+		sixdeskmess  1 "Depending on the number of points in the scan, this operation can take up to few minutes."
+		ii=0
+		while read tmpDir ; do
+		    taskid="htcondor${clusterID}.${ii}"
+		    Runnam=$(sixdeskFromJobDirToJobName ${tmpDir} ${lbackcomp})
+		    updateTaskIdsCases $sixdeskjobs/jobs $sixdeskjobs/incomplete_jobs $taskid $Runnam
+		    let NsuccessSub+=1
+		    let ii+=1
+		done < ${sixdeskjobs}/${LHCDesName}.list		
+	    else
+		# Get the clusterid and write it to a file to maintain it for condor_transfer_data
+		clusterID=(${terseString})
+		clusterID=${clusterID[${#clusterID[@]}-1]}
+		clusterID=${clusterID//.}
+		echo ${clusterID} >> ${sixdeskwork}/htboincjobs/clusterids 
+		sixdeskmess -1 "clusterID  : ${clusterID}"
+		sixdeskmess -1 "${terseString}"
+
+		ii=0
+		while read tmpDir ; do		    
+#		    echo "removing "
+		    rm ${sixdesktrack}/$tmpDir/SixIn.zip
```

Comment:
I would stay with the name of the .zip from the .list (see later comment on dot_htboinc)